Explorer/Assets/Scripts/Global/Dynamic/GlobalWorldFactory.cs

```diff
@@ -176,6 +176,8 @@ public class GlobalWorldFactory
 
             staticContainer.GlobalWorldProxy.SetWorld(world);
 
+            sceneFactory.SetPlayerEntity(globalWorld.EcsWorld, playerEntity);
```

Comment:
I don't think this is right, we are passing 2 arguments to use them together at `RestrictedActionsAPIImplementation`.

I have 2 ideas:
- Creating and passing a new interface called `IGlobalWorldActions` which implements `void ForcePlayerReposition` and `void ForceCameraLookAt`
- Creating 2 Commands `IPlayerReposition` and `ICameraLookAt` with their proper `Execute` methods.

WDYT?

Reply:
I finally decided to follow your first suggestion. Now everything is much more encapsulated. Thanks!

You can take a look to the change here: https://github.com/decentraland/unity-explorer/pull/322/commits/00b7f4d6ba74648fe89a80e52d95cf00ee84fd54